### PR TITLE
Add contribution section to new plugin issue template

### PR DESCRIPTION
**Pull request by florisvdg**
https://github.com/1Password/shell-plugins/blob/26dea93b53be88ec7acd572b5e63a4a51f336122/.github/ISSUE_TEMPLATE/1-new-plugin-request.yml

### CODE REVIEW

**Review by SimonBarendse — APPROVED** [review]

**Review by hculea — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/1-new-plugin-request.yml [hunk 1]
```diff
@@ -30,6 +30,13 @@ body:
       placeholder: "For example: Authenticate the CLI using a Personal Access Token."
     validations:
       required: true
+  - type: checkboxes
+    id: contribution
+    attributes:
+      label: Contribution
+      description: Shell plugins are a community effort, so you're more than welcome to [build the plugin yourself](https://developer.1password.com/docs/cli/shell-plugins/contribute/)!
+      options:
+        - label: I'd be up for building the plugin myself.
```
Comment (florisvdg) [inline .github/ISSUE_TEMPLATE/1-new-plugin-request.yml:39]: Better?
```suggestion
        - label: I'd be up for building the plugin myself!
```

File: .github/ISSUE_TEMPLATE/1-new-plugin-request.yml [hunk 2]
```diff
@@ -30,6 +30,13 @@ body:
       placeholder: "For example: Authenticate the CLI using a Personal Access Token."
     validations:
       required: true
+  - type: checkboxes
+    id: contribution
+    attributes:
+      label: Contribution
+      description: Shell plugins are a community effort, so you're more than welcome to [build the plugin yourself](https://developer.1password.com/docs/cli/shell-plugins/contribute/)!
```
Comment (florisvdg) [inline .github/ISSUE_TEMPLATE/1-new-plugin-request.yml:37]: Better?
```suggestion
      description: Shell plugins are a community effort, so you're more than welcome to [build the plugin yourself](https://developer.1password.com/docs/cli/shell-plugins/contribute/)! We're here to help you out if needed.
```

**Review by jpcoenen — DISMISSED** [review]
All variations look good to me :+1:

File: .github/ISSUE_TEMPLATE/1-new-plugin-request.yml (hunk 2 above)
Comment (SimonBarendse) [reply .github/ISSUE_TEMPLATE/1-new-plugin-request.yml:37]: We need to be careful that this is not interpreted as a commitment that we'll be building it. You could consider a slight variation:

```suggestion
      description: Shell plugins are a community effort, so you're more than welcome to [build the plugin yourself](https://developer.1password.com/docs/cli/shell-plugins/contribute/)! We're here to help when you have any questions.
```

**Review by SimonBarendse — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/1-new-plugin-request.yml [hunk 3]
```diff
@@ -30,6 +30,13 @@ body:
       placeholder: "For example: Authenticate the CLI using a Personal Access Token."
     validations:
       required: true
+  - type: checkboxes
+    id: contribution
+    attributes:
+      label: Contribution
+      description: Shell plugins are a community effort, so you're more than welcome to [build the plugin yourself](https://developer.1password.com/docs/cli/shell-plugins/contribute/)! We're here to help when you have any questions.
```
[the remark this answers is not in the document]
Comment (florisvdg) [reply .github/ISSUE_TEMPLATE/1-new-plugin-request.yml:37]: I think that's fine because they're already in the process of contacting us.

**Review by SimonBarendse — APPROVED** [review]

**Pull request merged**